tasks/requirements.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Pre-taks | populate service ansible_facts
+  service_facts:
+
+- name: Pre-taks | Install pip (Debian)
+  become: yes
+  apt:
+    name:
+    - python-pip
+    - python3-pip
```

Comment:
we could rather autodetect the python version in used.

Reply:
well, yes but for new releases of the module :)